go.mod

```diff
@@ -3,24 +3,26 @@ module github.com/ElrondNetwork/elrond-proxy-go
 go 1.17
 
 require (
-	github.com/ElrondNetwork/elastic-indexer-go v1.2.30
-	github.com/ElrondNetwork/elrond-go v1.3.27
-	github.com/ElrondNetwork/elrond-go-core v1.1.18
-	github.com/ElrondNetwork/elrond-go-crypto v1.0.1
-	github.com/ElrondNetwork/elrond-go-logger v1.0.7
+	github.com/ElrondNetwork/elastic-indexer-go v1.2.44-0.20221006131929-a7c3dec53ba1
+	github.com/ElrondNetwork/elrond-go v1.3.45-0.20221020085243-978bf0f08a13
```

Comment:
should we use proper tags before merging?

Reply:
These are the commit hashes from feat branches on elrond-go/go-core. I don't think there is any tag for that